chpl/chpl-service/src/test/java/gov/heatlhit/chpl/validation/listing/CertifiedProductValidationTest.java

```diff
@@ -58,26 +58,34 @@
 
 /**
  * Tests of the Certified Product validator.
+ * 
  * @author alarned
  *
  */
 
-
-// This test class has a modified configuration to get the tests to work.  The method
-// CertificationResultsManagerImpl.getCertifiedProductHasAdditionalSoftware() does not 
-// work in the test environment, so we are overriding that method.  Since we are not 
-// testing that particular method with these tests, this should be OK.  To do this, we 
+// This test class has a modified configuration to get the tests to work. The
```

Comment:
What happened here? Definitely got harder to read. Maybe worth changing it to use the multi line format? 
`/**`
`* comments go here`
`* and sentences can wrap as necessary...`
`*/`